doc/examples/aggregated_distances.ipynb

```diff
@@ -0,0 +1,654 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "for t in range(h.max_t, h.max_t+1):\n",
+    "    ax = pyabc.visualization.plot_kde_2d(*h.get_distribution(m=0, t=t),\n",
+    "                                         \"theta1\", \"theta2\",\n",
```

Comment:
The plot legend is "$\\Theta$" therefore I would also plot as X and Y axis labels "$\\Theta_1$" and "$\\Theta_2$"
or change the legend name to "theta" (consistency)